scripts/behavior.js

```diff
@@ -371,7 +371,129 @@ function executeCommand(command, arguments, cooldown, message, settings, utils,
                 }).then(
                     callback(COMMAND_SUCCESS)
                 );
-                
+
+                break;
+
+            case "createrole":

```

Comment:
Please methodify those 4 commands i.
Code in the cases of the switch (as you can see) should be kept as little as possible so that this main "hub" function can serve as a summary of what the bot can do and will do. Ideally, a case: will just fire a function and .then() callback. 
See the ladderpool command or the fixbridge command for example.

Reply:
I expected this review and would have done that myself the next days anyways ;). The problem I see with this as that the .then() will always callback, getting the result of the command execution is not that easy in there, so I would have problems returning COMMAND_MISUSE. I could of course do this in the error handler of a promise (I bet there is one). .then(successFunc, errorFunc)